docs/UserGuideCS2101.md

```diff
@@ -0,0 +1,424 @@
+
+### Adding a question : `addq`
+
+Adds a question to the question list.
+This feature allows you to keep track of the students' questions.
+
+Format: `addq QUESTION_DESCRIPTION`
+
+Example:
+
+* `addq what is the difference between self-invocation and call-backs for sequence diagrams?`
+
+### Marking a question : `markq`
+
+Marks a question as important.
+This feature enables you to indicate on the user interface that this question is important and requires more 
+attention compared to the other questions.
+
+Format: `markq INDEX`
+
+* Marks the question at the specified INDEX as important.
+* The index refers to the index number shown in the displayed question list.
+* The index must be a positive integer 1, 2, 3, ...
+* The index must be within the number of questions in the question list. E.g. There are 4 questions. The possible
+  indexes are 1, 2, 3 and 4.
+
+Example:
+
+* `markq 1` marks the first question in the question list as important
+
+### Unmarking a question : `unmarkq`
+
+Marks a question as unimportant. (If the question was previously or mistakenly marked as important)
+
+Format: `unmarkq INDEX`
+
+* Marks the question at the specified INDEX as unimportant.
+* The index refers to the index number shown in the displayed question list.
+* The index must be a positive integer 1, 2, 3, ...
+* The index must be within the number of questions in the question list. E.g. There are 4 questions. The possible
+  indexes are 1, 2, 3 and 4.
+
+Example:
+
+* `unmarkq 1` marks the first question in the question list as unimportant
+
+### Deleting a question : `deleteq`
+
+Deletes a question in the question list.
+Once a question has been addressed, you can delete it from the list.
+
+Format: `deleteq INDEX`
+
+* The index refers to the index number shown in the displayed question list.
+* The index must be a positive integer 1, 2, 3, ...
+* The index must be within the number of questions in the question list. E.g. There are 4 questions. The possible
+  indexes are 1, 2, 3 and 4.
+
+Example:
+
+* `deleteq 1` deletes the first question from the question list
+
+### Tutorials
+Tutorials contain a `group number`, `content` to be taught, and its `date and time`. 
+
+
+### Adding a tutorial : `addtut`
+
+Adds a tutorial to the tutorial list.
+
+Format: `addtut g/GROUP_NUMBER c/CONTENT t/DATE TIME`
+
+* The format of the date must be in the YYYY-MM-DD.
+* The format of the time must be in 24h format HHmm.
+
+Example:
+
+* `addtut g/T08 c/UML diagrams t/2022-10-01 1400`
+
+### Deleting a tutorial : `deletetut`
+
+Deletes a tutorial in the tutorial list.
+
+Format: `deletetut INDEX`
+
+* Deletes the tutorial at the specified INDEX.
+* The index refers to the index number shown in the displayed tutorial list.
+* The index must be a positive integer 1, 2. 3, … .
+*
+
+Example:
+
+* `deletetut 1`deletes the first tutorial from the tutorial list
+
+### Marking a tutorial: `marktut`
+
+Marks content in the tutorial as done.
```

Comment:
Marks the tutorial as done, not just the content.

Reply:
done